From 6ad752f91756aa252173f690c0b20d74b395f65f Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Wed, 20 Dec 2017 17:54:33 +0800
Subject: [PATCH 3140/5242] MLK-17290-06 arm64: dts: gpio: add mipi csi SS
 gpio clock and power domain

commit  71cf8842a11e58f1dddb851eac026bb6db0f8887 from
https://source.codeaurora.org/external/imx/linux-imx.git

GPIO in MIPI CSI SS also has its related ipg clock and power
domain, add them.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Tested-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Reviewed-by: Frank Li <Frank.Li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index f00d9e0..7841904 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1918,6 +1918,11 @@
 		interrupt-parent = <&irqsteer_csi>;
 		gpio-controller;
 		#gpio-cells = <2>;
+		interrupt-controller;
+		#interrupt-cells = <2>;
+		power-domains = <&pd_mipi_csi>;
+		clocks = <&clk IMX8QXP_CSI0_IPG_CLK_S>;
+		clock-names = "ipg";
 	};
 
 	gpu_3d0: gpu@53100000 {
-- 
1.7.9.5

